uwsgi-example/hooks/run

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+
+# We have to call 'hooks/run' so that we pickup its workarounds instead of
+# calling 'uwsgi' directly, or we could duplicate the workarounds here.
+exec "$(hab pkg path tvaughan/uwsgi)/hooks/run"                               \
```

Comment:
Because `uswgi/` and `uwsgi-example/` are two different directories at the same level I had to create two separate studios. I couldn't find a way to share the application that I build in the `uwsgi/` studio with the `uwsgi-example/` studio. I had to push the uwsgi application to a tvaughan origin https://app.habitat.sh/#/origins/tvaughan which would then be downloaded into the `uwsgi-example/` studio. What did I miss? What's a better way to do this? Should `uwsgi-example/` be a subdirectory under `uwsgi/` or something else?